GitHub Actions failing at installation of Node.js

A few days ago, some of my CI checks randomly failed at the `install moodle-plugin-ci` step. After a few retries, all checks finally passed. In the meantime, things have become worse. Currently, all checks are constantly failing. Some other developers reported the same in the dev chat.

```
 0/9 [>---------------------------]   0% < 1 sec [Starting install]
 2/9 [======>---------------------]  22% 15 secs [Moodle assets]
 3/9 [=========>------------------]  33% 15 secs [Install plugins]
 4/9 [============>---------------]  44% 16 secs [Installing Node.js]
In VendorInstaller.php line [12](https://github.com/PhilippImhof/moodle-quiz_essaydownload/actions/runs/10297980501/job/28511977093#step:6:13)9:
                                
  Node.js installation failed.  
                                
install [--moodle MOODLE] [--data DATA] [--repo REPO] [--branch BRANCH] [--plugin PLUGIN] [--db-type DB-TYPE] [--db-user DB-USER] [--db-pass DB-PASS] [--db-name DB-NAME] [--db-host DB-HOST] [--db-port DB-PORT] [--not-paths NOT-PATHS] [--not-names NOT-NAMES] [--extra-plugins EXTRA-PLUGINS] [--no-init] [--no-plugin-node] [--node-version NODE-VERSION]
Error: Process completed with exit code 1.
```

--- a/docs/CHANGELOG.md
+++ b/docs/CHANGELOG.md
@@ -9,6 +9,9 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 The format of this change log follows the advice given at [Keep a CHANGELOG](https://keepachangelog.com).
 
 ## [Unreleased]
+### Changed
+- Fixed nvm loading issue caused by upstream regression.
+
 ## [4.5.3] - 2024-07-05
 ### Added
 - Support for version 4.4 of the app, that uses new defaults and Chrome (Selenium 4) version.

--- a/src/Installer/VendorInstaller.php
+++ b/src/Installer/VendorInstaller.php
@@ -120,7 +120,7 @@ class VendorInstaller extends AbstractInstaller
         }
 
         $nvmDir  = getenv('NVM_DIR');
-        $cmd     = ". $nvmDir/nvm.sh && nvm install && nvm use && echo \"NVM_BIN=\$NVM_BIN\"";
+        $cmd     = ". $nvmDir/nvm.sh; nvm install && nvm use && echo \"NVM_BIN=\$NVM_BIN\"";
 
         $process = $this->execute->passThroughProcess(
             Process::fromShellCommandline($cmd, $this->moodle->directory, null, null, null)

